package.json

```diff
@@ -28,7 +28,7 @@
   },
   "homepage": "https://github.com/mzabriskie/react-disqus-thread",
   "dependencies": {
-    "react": "^0.12.0"
+    "react": "0.12.x - 1.x"
```

Comment:
Also changed this since otherwise the React versions seem to conflict and prevent the component from working.